backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/VerificationGraphQLHandler.kt

```diff
@@ -0,0 +1,22 @@
+package app.ehrenamtskarte.backend.verification.webservice
+
+import app.ehrenamtskarte.backend.common.webservice.GraphQLHandler
+import com.expediagroup.graphql.SchemaGeneratorConfig
+import com.expediagroup.graphql.TopLevelObject
+import com.expediagroup.graphql.toSchema
+import graphql.GraphQL
+
+
+class VerificationGraphQLHandler : GraphQLHandler(graphQL) {
+    companion object {
+        private val config = SchemaGeneratorConfig(supportedPackages = listOf("app.ehrenamtskarte.backend.verification.webservice.schema"))
+        private val queries = listOf<TopLevelObject>(
+            // TODO add query to check if card exists
+        )
+        private val mutations = listOf<TopLevelObject>(
+            // TODO add addCard mutation
+        )
+        val graphQLSchema = toSchema(config, queries, mutations)
+        val graphQL = GraphQL.newGraphQL(graphQLSchema).build()!!
```

Comment:
Please add another run config to generate the schema. (or create an issue)

Reply:
Should already be there